stock_picking_mixin/models/stock_picking_mixin.py

```diff
@@ -0,0 +1,37 @@
+# © 2021 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3.0 (http://www.gnu.org/licenses/lgpl-3.0.html)
+
+from odoo import fields, api, models
+
+
+class StockPickingMixin(models.AbstractModel):
+    _name = "stock.picking.mixin"
```

Comment:
The current abstract model should be renamed, and the functionality isolated within this addon:
```suggestion
    _name = "stock.picking.state.mixin"
```
Then, adding new abstract functionalities to this addon becomes adding new and independent models